magicbell-rails.gemspec

```
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name        = 'magicbell-rails'
-  spec.version     = '0.3.1'
+  spec.version     = '0.3.2'
```

it's an additional feature and not a bug fix, so it's a minor update. Based on https://semver.org/ 

```suggestion
  spec.version     = '0.4.0'
```

---

Ahh! very bad habit of mine I need to get out of, thanks